epoch1/cuda/ee_mumu/SubProcesses/Makefile

```diff
@@ -70,19 +70,19 @@ $(LIBDIR)/lib$(MODELLIB).a:
 	$(MAKE) -C ../../src $(MAKEDEBUG)
 
 gcheck.o: gcheck.cu *.h ../../src/*.h ../../src/*.cu
-	$(NVCC) $(CPPFLAGS) $(CUFLAGS) -c $< -o $@
```

Comment:
This violates the [Makefile conventions](https://www.gnu.org/software/make/manual/html_node/Command-Variables.html), which would allow users to add a C++ flag without breaking the makefile.

Reply:
Hi Stephan, thanks but sorry I do not get it. What are you suggesting exactly? Are you suggesting that CPPFLAGS should be added back? Thanks Andrea

Reply:
Ah yes I imagine that this is waht you meant. OK will do. Thanks!
Andrea

Reply:
Ok done, thanks